### PR TITLE
Lesson 5

### PR DESCRIPTION
**Pull request by Irina-pi**

### CODE REVIEW

File: styles/fonts.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* cyrillic-ext */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+  src: local("Roboto Regular"), local("Roboto-Regular") url('./.../fonts/roboto-regular-cyrillic-ext.woff2') format('woff2');
```
Comment (5tronciy) [inline styles/fonts.css:7]: Wrong path in url()

File: styles/styles.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  max-width: 1440px;
```
Comment (5tronciy) [inline styles/styles.css:13]: Not sure, that you need this limit. It looks good without it on wide screens.

File: styles/styles.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  font-weight: 400;
+  line-height: 28px;
+  font-size: 24px;
+}
+
+.footer {
+  background-color: #10367c;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 1440px;
```
Comment (5tronciy) [inline styles/styles.css:63]: The same as in the row 13

File: styles/styles.css [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+.line {
+  background-color: #f5bd41;
+  border: 1px solid #f5bd41;
+  width: 100px;
+  height: 1px;
+  align-self: center;
+  margin-top: 20px;
+}
```
Comment (5tronciy) [inline styles/styles.css:38]: This line is thicker in design.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Looks better than at last lesson.

File: styles/styles.css (hunk 2 above)
Comment (Irina-pi) [reply styles/styles.css:13]: Ok. Thank you!

File: styles/styles.css (hunk 3 above)
Comment (Irina-pi) [reply styles/styles.css:63]: Ok.

File: styles/styles.css (hunk 4 above)
Comment (Irina-pi) [reply styles/styles.css:38]: Maked border 2px. now matches.

Comment (Irina-pi) [reply styles/styles.css:38]: Ok. corrected on " url(./../fonts/roboto-regular-latin-ext.woff2) ". Anothe too.

File: styles/fonts.css (hunk 1 above)
Comment (Irina-pi) [reply styles/fonts.css:7]: Ok. corrected on " url(./../fonts/roboto-regular-latin-ext.woff2) ". Anothe too.

**Pull request merged**